apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -91,7 +91,8 @@ function use_font_smooth_antialiased() {
 		return true;
 	}
 
-	return apply_filters( 'a8c_use_font_smoothing_antialiased', false );
+	// Open anti-aliasing for all by default.
+	return apply_filters( 'a8c_use_font_smoothing_antialiased', true );
```

Comment:
Now that we don't need access to is-a11n logic I think we should simplify things and just remove this filter. Doesn't have to be this PR.

Reply:
Was second guessing whether to simplify this further, thanks for the vote of confidence.